frontend/src/Pages/ResourcesManagement.tsx

```diff
@@ -217,13 +319,12 @@ export default function ResourcesManagement() {
                     <div className="flex justify-center">
                         <button
                             className="btn btn-primary btn-sm flex items-center justify-center space-x-2 text-white"
-                            onClick={(e) => {
-                                void updateFinalState(e);
-                            }}
+                            //eslint-disable-next-line
```

Comment:
Can we resolve this so eslint-disable-next-line isn't necessary? If we need it, please include a comment explaining why we need it. 

Look at the previous implementaiton which used `void updateFinalState(e)`. This is consistent with how we do it in the rest of the codebase.